package/gluon-wifi-encrypt/Makefile

```diff
@@ -0,0 +1,13 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gluon-wifi-encrypt
```

Comment:
I know we often use the term "WiFi" like in the wifi-config package. The correct technical term is WLAN as WiFi is a brand. Yes, we say "googling" instead of "searching the internet", but that should just be slang.
Furthermore "encrypt" is too general as we can use other means of encryption like SAE.
Thus I'd like to call this package "gluon-wlan-encryption-psk"